reckon-core/src/main/groovy/org/ajoberstar/reckon/core/Versioners.java

```diff
@@ -19,77 +19,79 @@
 
 import java.util.Objects;
 
-/**
- * Companion functions for creating common versioners.
- */
+/** Companion functions for creating common versioners. */
 public final class Versioners {
-    private Versioners() {
-        throw new AssertionError("Cannot instantiate this class.");
-    }
+  private Versioners() {
+    throw new AssertionError("Cannot instantiate this class.");
+  }
 
-    /**
-     * A Version 0.0.0
-     */
-    public static final Version VERSION_0 = Version.forIntegers(0, 0, 0);
+  /** A Version 0.0.0 */
+  public static final Version VERSION_0 = Version.forIntegers(0, 0, 0);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.ajoberstar.com/coding_rules#rule_key=squid%3AS1213)